components/badge/badge.css

```diff
@@ -0,0 +1,92 @@
+.avatar-badge {
+  position: relative;
+  border-radius: 50%;
+}
+
+.badge-avatar-xs {
+  width: 3rem;
+  height: 3rem;
+  font-size: 2rem;
+}
+
+.badge-avatar-m {
+  width: 5rem;
+  height: 5rem;
+  font-size: 2rem;
+}
+
+.badge-text-avatar {
+  background-color: var(--secondary-clr);
+  color: var(--white);
+  width: 7rem;
+  height: 7rem;
+  font-size: 2.5rem;
+}
+
+.badge-xs {
+  left: 2.3rem;
+  bottom: 2.3rem;
+}
+
+.badge-m {
+  left: 3.9rem;
+  bottom: 4rem;
+}
+
+.badge-lg {
+  left: 6.2rem;
+  bottom: 5.2rem;
+}
+
+.badge-icon {
+  font-size: 2rem;
+  position: relative;
+  color: var(--black);
+}
+
+.icon-badge-pos {
+  left: 1.7rem;
+  bottom: 1.5rem;
+}
+
+.badge {
+  position: absolute;
+  width: 1rem;
+  height: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 50%;
+}
+
+.number-badge {
+  position: absolute;
+  width: 1.5rem;
+  height: 1.5rem;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 50%;
+  color: var(--white);
+}
+
+.font-size-icon {
+  font-size: 0.7rem;
+}
+
+.font-size-badge {
+  font-size: 0.9rem;
+}
+
+.red-badge {
+  background-color: var(--red);
+}
+
+.green-badge {
+  background-color: var(--green);
+}
+
+.border-white {
+  border: 1px solid var(--white);
+}
```

Comment:
Seems like a generic class.

Reply:
Yes, nice find.. will add this to utility